ARCHITECTURE.md

```diff
@@ -41,4 +42,27 @@ For more details about the models we use in Scanner see web-monitoring-db's [API
+|B| IA's Wayback Machine is queried for imports | [web-monitoring-processing](https://github.com/edgi-govdata-archiving/web-monitoring-processing) - this [PR](https://github.com/edgi-govdata-archiving/web-monitoring-processing/pull/174) | Data is pulled from the IA API, formatted, and [POST](https://api.monitoring.envirodatagov.org/#/imports/post_imports_)ed to [-db](https://github.com/edgi-govdata-archiving/web-monitoring-db) | Runs on a cron job, formats per [GH issue comment](https://github.com/qri-io/walk/issues/16#issuecomment-437785099) |
+|C| New metadata arrives at [-db](https://github.com/edgi-govdata-archiving/web-monitoring-db) | Scripts running on cron jobs that do ETL via scrapes or APIs (These scripts: [Versionista scraper](https://github.com/edgi-govdata-archiving/web-monitoring-versionista-scraper), [Wayback Machine importer](https://github.com/edgi-govdata-archiving/web-monitoring-processing))| [POST](https://api.monitoring.envirodatagov.org/#/imports/post_imports_) to [/api/v0/imports](https://api.monitoring.envirodatagov.org/#/imports/post_imports_) as JSON array or newline-delimited JSON stream (stream preferred) | Contains:<ul><li> a URL for where to retrieve the raw response body of the version</li><li> a SHA-256 hash of that data</li></ul> Example data in [GH issue comment](https://github.com/qri-io/walk/issues/16#issuecomment-437785099)|
+|D| Determination is made whether or not to download the data | [-db](https://github.com/edgi-govdata-archiving/web-monitoring-db) (specifics useful?) | If the URL is in an acceptable, publicly readable location |  Acceptable location checked [per .env](https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/30d976676935f5f621e18285016a137c8b55a0d8/.env.example#L26-L27) |
+|    ↳ E1| URL for raw data is stored  | [-db](https://github.com/edgi-govdata-archiving/web-monitoring-db) (specifics useful?) | [-db](https://github.com/edgi-govdata-archiving/web-monitoring-db) stores URL (where? Presumably some file in our S3 bucket?) | Happens if [-db](https://github.com/edgi-govdata-archiving/web-monitoring-db) has determined not to download the data |
+|    ↳ E2| Raw data is verified and stored on URL we store and maintain | [-db](https://github.com/edgi-govdata-archiving/web-monitoring-db) (specifics useful?) | Raw response data downloaded from the URL, verified against SHA-256 hash of the data from initial POST, and stores it (in production) in a public S3 bucket | Happens if [-db](https://github.com/edgi-govdata-archiving/web-monitoring-db) has determined to download the data |
```

Comment:
Just realized it might make sense to point out that in E1 we didn’t touch the metadata’s `uri` field, and in E2, we replace the metadata’s `uri` with the URL to wherever we stored the raw version response body.